### ISSUE TITLE
Issue migrating with version 2.5.0

### ISSUE THREAD

**Issue by jproffitt** [divio/djangocms-text-ckeditor#245]
I just upgraded to version 2.5.0, removed the entry in `MIGRATION_MODULES`, and ran migrate. I got this strange error. Everything worked fine before upgrading.

    django.core.exceptions.ImproperlyConfigured: Application labels aren't unique, duplicates: djangocms_text_ckeditor

**Comment by yakky** [comment]
This appears when an application is present multiple times in ``INSTALLED_APPS``. What's your setting?

**Comment by jproffitt** [comment]
That's not the case:

    >>> import collections
    >>> from django.conf import settings
    >>> collections.Counter(settings.INSTALLED_APPS)['djangocms_text_ckeditor']
    1

I am using Django version 1.7.8. If I install `djangocms-text-ckeditor==2.4.3`, and run migrate, there are no issues at all. After installing version 2.5.0, and changing nothing at all other than removing the relevant item in `MIGRATION_MODULES`, I get the error when migrating. I even tried uninstalling completely, and making sure all .pyc fils are gone, then installing again.

**Comment by yakky** [comment]
Which django CMS version are you using?

**Comment by jproffitt** [comment]
3.1.0

**Comment by ooskr** [comment]
I did this to get it to work with Django 1.7.8 and Django CMS 3.1.0 

https://docs.djangoproject.com/en/1.7/ref/applications/#for-application-authors

**Comment by yakky** [comment]
@ooskr This has been tested on Django 1.7 without the AppConfig just fine, I wonder why it's required.
Anyway @jproffitt could you test this branch https://github.com/yakky/djangocms-text-ckeditor/archive/django_app.zip? (``pip install https://github.com/yakky/djangocms-text-ckeditor/archive/django_app.zip`` will install the updated package)

**Comment by jproffitt** [comment]
Well I get a different error now, but thats just because `apps.TextCkeditorConfig.name` is misspelled. Once I correct that spelling, migrating works like normal!

Just to double check, I installed 2.5.0 again, and I got that error again. But I noticed another migration shows up in the migration folder when I install 2.5.0, `0002_rename_plugin.py`, which is a south migration. There's actually three migration folders: (`migrations`, `migrations_django`, and `south_migrations`). 

So I bet thats the reason it crashes. Somehow the migration folders got all mixed up in 2.5.0. In the django_app branch, the migration folders are correct again.
